container-templates/dockerfile/.devcontainer/devcontainer.json

```diff
@@ -1,10 +1,23 @@
 {
-    // "See https://aka.ms/vscode-remote/devcontainer.json for format details."
-    "name": "Dev Container Definition Template - Dockerfile",   
-    "dockerFile": "Dockerfile",
-    "appPort": 3000,
-    "extensions": [
-        "mutantdino.resourcemonitor"
-    ],
-    "runArgs": []
+	// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+	"name": "Dev Container Definition Template - Dockerfile",
+
+	// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+	"dockerFile": "Dockerfile",
+
+	// Use the optional 'appPort' property to expose ports not already in your Dockerfile
```

Comment:
We don't automatically expose ports `EXPOSE`d in the Dockerfile yet. So this is promising a little too much.